contracts/src/v0.1/interfaces/IAccount.sol

```diff
@@ -129,4 +143,25 @@ interface IAccount {
      * @param to - Where to send the remaining KLAY to
      */
     function cancelAccount(address to) external;
+
+    function getAccountDetail() external view returns (uint256, uint256, uint256, uint256);
+
+    function getSubscriptionPaid() external view returns (bool);
+
+    function updateAccountDetail(
+        uint256 startDate,
+        uint256 period,
+        uint256 reqPeriodCount,
+        uint256 subscriptionPrice
+    ) external;
+
+    function setSubscriptionPaid() external;
+
+    function isValidReq() external view returns (bool);
+
+    function getFeeRatio() external view returns (uint256);
+
+    function setFeeRatio(uint256 disCount) external;
+
+    function increaseReqCount() external;
```

Comment:
`Account` contract defines a state variable `sReqCount` that is increased when we `chargeFee`. In the new payment system, it is beneficial only to `KLAY_REGULAR` and `KLAY_DISCOUNT` account types. The function above `increaseReqCount` is called only for `KLAY_SUBSCRIPTION` and `FIAT_SUBSCRIPTION`. To avoid confusion between `sReqCount` and `increaseReqCount`, I would suggest to rename the `increaseReqCount` to `increaseSubReqCount` or some other name if more appropriate. Similarly, `sReqCountHistory` from `Account` contract could be renamed to `sSubReqCountHistory` for an easier understanding.

https://github.com/Bisonai/orakl/blob/423084bba51c9ba136ea26088fb81878991d39a9/contracts/src/v0.1/Account.sol#L230